libraries/botbuilder/src/inspectionMiddleware.ts

```diff
@@ -12,6 +12,17 @@ import { Middleware, TurnContext, BotState, StatePropertyAccessor, UserState, Co
 /** @private */
 class TraceActivity {
 
+    public static fromCommand(command: string): Partial<Activity> {
```

Comment:
Might want to change this function name to `makeCommandActivity` to align with [what is currently in the 4.4 branch](https://github.com/microsoft/botbuilder-js/blob/4.4/libraries/botbuilder/src/inspectionMiddleware.ts#L15)

Reply:
updated function name.